worker/backup_processor.go

```diff
@@ -157,14 +181,8 @@ func (pr *BackupProcessor) WriteBackup(ctx context.Context) (*pb.BackupResponse,
 			return false
 		}
 
-		// Backup type keys in every group.
-		if parsedKey.IsType() {
-			return true
-		}
-
-		// Only backup schema and data keys for the requested predicates.
-		_, ok := predMap[parsedKey.Attr]
-		return ok
```

Comment:
We should have this back. Otherwise, we will also backup the stale data. 
Also, handle this case in schema.

Reply:
Nice catch. Fixed it.